docs/how-to/visualization/graphs.qmd

```diff
@@ -0,0 +1,84 @@
+# GraphViz + Ibis
+
+Ibis supports visualizing an expression as a [directed
+graph](https://en.wikipedia.org/wiki/Directed_graph) using
+[GraphViz](https://graphviz.org/).
+
+To get started, make sure you've installed the necessary dependencies.
+
+```sh
+$ pip install 'ibis-framework[duckdb,examples,graphviz]'
+```
+
+::: {.callout-note collapse="true"}
+## Replace `duckdb` with your backend of choice
+
+For instance, you can replace `duckdb` with `snowflake` if you want to use the
+[Snowflake backend](../../backends/snowflake.qmd).
+:::
+
+
+Let's run through a few examples.
+
+First we'll import the things we need.
+
+```{python}
+import ibis
+
+from ibis import _
+from ibis.expr.visualize import to_graph
+```
+
+Now we can visualize an expression graph.
+
+Here's a call to `select`.
+
+```{python}
+t = ibis.examples.penguins.fetch()
+expr = t.select(lowered=_.species.lower())
+to_graph(expr)
+```
+
+The way to read the graph is from top to bottom.
+
+* The top of the graph contains Ibis operations with no dependents.
+* The bottom of the graph contains Ibis operations with no dependencies.
```

Comment:
this reads slightly off to me -- "top to bottom", then bullet points for top and bottom followed by the things in the middle -- in my mind would be top, then middle, then bottom, but not a big deal IMO

Reply:
Yeah, good point. I can rearrange.

Reply:
Fixed!